File: JeongEon8/02/[PGS - Lv2] 멀리 뛰기.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int n) {
    long long answer = 0;
    long long run[2000];
    
    run[0] = 0;
    run[1] = 1;
    run[2] = 2;
    
    for(int i = 3; i <= n; i++){
        run[i] = (run[i-1] + run[i-2]) %1234567;
    }
        
    answer = run[n];
    
    return answer;
}

File: 9월 2주차/hyuksanglee/02/[PGS - Lv2] 멀리 뛰기.java
class Solution {
    
    
    public long solution(int n) {
        int answer =0;
        long[] dp = new long[n+1];
        
        
        for(int i =1; i<= n; i++){
            if(i==1)dp[1] =1;
            else if(i ==2)dp[2] =2;
            else dp[i] = (dp[i-1]+dp[i-2])% 1234567;
        }
        return dp[n] ;
    }
    

}
